src/iconid.rs

```diff
@@ -187,7 +187,7 @@ fn apply_location_based_substitution(
 /// - A glyph is allowed to be assigned to multiple codepoints.
 /// - A glyph with a PUA and non-PUA codepoint is considered as single character icon and will be returned in the result.
 ///
-pub fn get_icons(font: &FontRef) -> Result<Vec<Icon>, IconResolutionError> {
+pub fn icons(font: &FontRef) -> Result<Vec<Icon>, IconResolutionError> {
```

Comment:
I don't mind this but note that it isn't quite what was suggested. The suggestion was to add a trait, implemented for FontRef, so we can say things like `font.icons()` instead of `icons(font)`. The former reads left-to-right and is consitent with how ligatures() was done.